libultraship/Makefile

```diff
@@ -17,7 +17,6 @@ WARN := -Wall -Wextra -Werror \
 	-Wno-unused-function \
 	-Wno-parentheses \
 	-Wno-narrowing \
-	-Wno-error=stringop-overflow \
```

Comment:
Clang on linux fails without this.

Reply:
what version of clang? what's the error it says?

Reply:
Clang++-11

```
In function ‘memcpy’,
    inlined from ‘aResampleImpl’ at libultraship/mixer.c:223:9:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:34:10: error: writing 16 bytes into a region of size 0 [-Werror=stringop-overflow=]
   34 |   return __builtin___memcpy_chk (__dest, __src, __len, __bos0 (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
libultraship/mixer.c: In function ‘aResampleImpl’:

```

https://pastebin.com/raw/KF6bd9GE

[FULL BUILD](https://github.com/qurious-pixel/Shipwright-linux/actions/runs/2641306318)

Reply:
shoudl be addressed now